src/api/api.test.tsx

```diff
@@ -0,0 +1,51 @@
+import itemsData from 'data/items';
+import { getItems } from './api';
+
+describe('getItems', () => {
```

Comment:
I recommend clearing all mocks https://jestjs.io/docs/jest-object#jestclearallmocks after each test to make sure you don't affect other test cases with this mocking